manifests/07-downloads-helm.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleCLIDownload
+metadata:
+  name: helm-download-links
+spec:
+  description: |
+    Helm 3 is a package manager for Kubernetes applications which enables defining,
+    installing and upgrading applications packaged as Helm Charts.
+    Helm 3 is a <a href="https://access.redhat.com/support/offerings/techpreview">Technology Preview</a> feature on OpenShift 4.
```

Prefer markdown-style links

```suggestion
    Helm 3 is a [Technology Preview](https://access.redhat.com/support/offerings/techpreview) feature on OpenShift 4.
```

---

Looks like the link still needs to be updated to markdown style `Helm 3 is a [Technology Preview](https://access.redhat.com/support/offerings/techpreview) feature on OpenShift 4.`

---

done.